master/internal/telemetry/reports.go

```diff
@@ -104,3 +104,18 @@ func ReportUserCreated(system *actor.System, admin, active bool) {
 		"active": active,
 	})
 }
+
+// ReportResourcePoolCreated reports that a resource pool has been created.
+func ReportResourcePoolCreated(
+	system *actor.System,
+	poolName, schedulerType,
+	fittingPolicy string,
+	preemptionEnabled bool,
+) {
+	report(system, "resource_pool_created", map[string]interface{}{
```

Comment:
Can we add a version number to this message so that it's easier to add new fields in the future?

Reply:
good idea, added it in.